proposals/coredns.adoc

```diff
@@ -0,0 +1,82 @@
+
+*Infrastructure requirements (CI / CNCF Cluster)*: N/A
+
+*Issue tracker*: https://github.com/miekg/coredns
+
+*Website*: https://coredns.io
+
+*Release methodology and mechanics*: github master
+
+*Social media accounts*: Twitter: @corednsio
+
+*Existing sponsorship*: Infoblox contributing developer time to implement CoreDNS->Kubernetes integration component.
+
+*External Dependencies*
+
+* Go package: beorn7/perks (MIT https://github.com/beorn7/perks/blob/master/LICENSE)
+* Go package: coreos/etcd (ASLv2 https://github.com/coreos/etcd/blob/master/LICENSE)
+* Go package: flynn/go-shlex (ASLv2 https://github.com/flynn-archive/go-shlex/blob/master/COPYING)
+* Go package: fsnotify/fsnotify (BSD https://github.com/fsnotify/fsnotify/blob/master/LICENSE)
+* Go package: golang/protobuf (BSD https://github.com/golang/protobuf/blob/master/LICENSE)
+* Go package: hashicorp/go-syslog (MIT https://github.com/hashicorp/go-syslog/blob/master/LICENSE)
+* Go package: matttproud/golang_protobuf_extensions (ASLv2 https://github.com/matttproud/golang_protobuf_extensions/blob/master/LICENSE 
+* Go package: miekg/dns (BSD https://github.com/miekg/dns/blob/master/LICENSE)
+* Go package: patrickmn/go-cache (MIT https://github.com/patrickmn/go-cache/blob/master/LICENSE)
+* Go package: prometheus/client_golang (ASLv2 https://github.com/prometheus/client_golang/blob/master/LICENSE)
+* Go package: prometheus/client_model (ASLv2 https://github.com/prometheus/client_model/blob/master/LICENSE)
+* Go package: prometheus/common (ASLv2 https://github.com/prometheus/common/blob/master/LICENSE)
+* Go package: prometheus/procfs (ASLv2 https://github.com/prometheus/procfs/blob/master/LICENSE)
+* Go package: ugorji/go (MIT https://github.com/ugorji/go/blob/master/LICENSE)
+* Go package: xenolf/lego (MIT https://github.com/xenolf/lego/blob/master/LICENSE)
+* Go package: golang/x/crypto (BSD https://github.com/golang/crypto/blob/master/LICENSE)
+* Go package: golang/x/net (BSD https://github.com/golang/net/blob/master/LICENSE)
+* Go package: golang/x/sys (BSD https://github.com/golang/sys/blob/master/LICENSE) 
+* Go package: natefinch/lumberjack.v2 (MIT https://github.com/natefinch/lumberjack/blob/v2.0/LICENSE) 
+* Go package: square/go-jose.v1 (ASLv2 https://github.com/square/go-jose/blob/master/LICENSE)
+* Kubernetes (for CoreDNS -> kubernetes integration) (ASLv2 https://github.com/kubernetes/kubernetes/blob/master/LICENSE) 
+
+*Statement on alignment with CNCF mission*:
+ 
+CoreDNS is a focused lightweight DNS server. A microservice philosophy guides the internal design of CoreDNS. Individual DNS functions are provided by discrete, composable plugins that are enabled via runtime configuration.
+CoreDNS can be thought of as a DNS protocol head that can be configured to front various backend data sources. A flexible DNS server is a necessary component to provide “Naming and Discovery” services to containers running in the CNCF distributed system services environment.
```

Comment:
A flexible DNS server is a necessary component in providing “Naming and Discovery” services to containers running in a cloud-native distributed environment.